File: sorting algorithm/Bubblesort.java
public class Bubblesort {

    int z = 0;

    void bsort(int[] arr){
        for (int i= 0; i < arr.length;i++){
            for (int j = 0; j < arr.length-1;j++){
                if (arr[j] > arr[j+1]){
                    int temp = arr[j+1];
                    arr[j+1] = arr[j];
                    arr[j] = temp;
                }
            }
        }

    }
    public static void main(String[] args) {

        Bubblesort bs = new Bubblesort();

        int[] a = {2,1,4,3,5,7};

        bs.bsort(a);

        for (int i : a){
            System.out.print(i+"  ");
        }

        
        
    }
    
}

File: sorting algorithm/bubble_sort.py
#BUBBLE SORT IMPLEMENTED IN PYTHON



def bubblesort(arr):
    

    for i in range(len(arr)):
        y = True
        for j in range(len(arr)-1):

            if arr[j] > arr[j+1]:
                temp = arr[j+1]
                arr[j+1] = arr[j]
                arr[j] = temp
                y = False
            
        if (y):
             break
    print(arr)


a = [2,1,4,3,5,7,1,2,5,6,7,8,5,2,3]

bubblesort(a)


"""
# Output

    [1, 1, 2, 2, 2, 3, 3, 4, 5, 5, 5, 6, 7, 7, 8]
"""





